six/modules/c++/six.sidd/include/six/sidd/SFA.h

```diff
@@ -101,6 +116,17 @@ struct SFAPoint : public SFAGeometry
         return new SFAPoint(*this);
     }
 
+    virtual bool equalTo(const SFATyped& rhs) const
```

Comment:
So what if you made an `operator==` that took in an `SFAPoint` and then still had the `equalTo()` but all it had to do was call `operator==` after the cast?  Same deal for all of these.